app/views/items/_form.html.erb

```diff
@@ -1,18 +1,21 @@
-
 <%= form_for @item do |f| %>
-  <%= f.text_field :name, placeholder: 'Name' %>
-  <%= f.date_field :date, placeholder: 'Date' %>
-  <%= f.text_field :description, placeholder: 'Description' %>
-  <%= f.number_field :price, placeholder: 'Price', min: 0, step: 0.01 %>
-  <%= f.submit 'Submit' %>
-   <div>
-      <select name='facility_id'>
-        <option>
-          <% @facilities.each do |f| %>
-            <option value="<%= f.id %>"><%= f.name %></option>
-          <% end %>
-        </option>
-      </select>
-      <%= link_to  "Add Facility",new_facility_path %>
-    </div>
-<% end %> 
+  <div class="p-2"><%= f.text_field :name, placeholder: 'Name' %></div>
+  <div class="p-2"><%= f.date_field :date, placeholder: 'Date' %></div>
+  <div class="p-2"><%= f.text_field :description, placeholder: 'Description' %></div>
+  <div class="p-2"><%= f.number_field :price, placeholder: 'Price', min: 0, step: 0.01 %></div>
+  <div class="p-2">
+     <select name='facility_id'>
+       <option>
+         <% @facilities.each do |f| %>
+           <option value="<%= f.id %>", placeholder:"Facility"><%= f.name %></option>
+         <% end %>
+       </option>
+     </select>
```

Comment:
Check out this Stack Overflow post on [how to create a dropdown in Rails](https://stackoverflow.com/questions/14113057/how-to-have-a-drop-down-select-field-in-a-rails-form).